src/Renci.SshNet/Session.cs

```diff
@@ -905,7 +905,13 @@ internal void WaitOnHandle(WaitHandle waitHandle, TimeSpan timeout)
             switch (WaitHandle.WaitAny(waitHandles, timeout))
             {
                 case 0:
+#if NETSTANDARD1_3 || NETSTANDARD2_0
+                    System.Runtime.ExceptionServices.ExceptionDispatchInfo.Capture(_exception).Throw();
```

Comment:
Capture and throw shouldn't be on the same line. Capture should happen when the exception is created.

Reply:
You're right, my implementation is totally out to lunch. I'll get it fixed up shortly.

Reply:
I haven't heard that one before. You just made my day.

Reply:
Is it OK if I update the language version of the project to `<LangVersion>latest</LangVersion>`? It's currently at C#5 but I'd like to use null propagating operators in C#6 to simplify some of my changes.

Reply:
Oh boy, I'd love to see the code modernized to never practices including null propagation and string interpolation. I am not a repo owner however, and don't know where they stand on that.

Reply:
Fixed